TrafficCapture/build.gradle

```diff
@@ -9,4 +10,15 @@ allprojects {
     test {
         jvmArgs '-ea'
     }
+
+    jacocoTestReport {
+        reports {
+            xml.required = true
+            xml.destination file("${buildDir}/reports/jacoco/test/jacocoTestReport.xml")
+        }
+    }
+}
+
+test {
```

Comment:
What are lines 22-24 doing? I didn't see any usage of this

Reply:
They are causing the `jacocoTestReport` task to run at the end of the test task, which means that's when the coverage will be available.